pkg/controller/vitesscluster/reconcile_keyspaces.go

```diff
@@ -189,6 +189,9 @@ func updateVitessKeyspaceInPlace(key client.ObjectKey, vtk *planetscalev2.Vitess
 	// Update labels, but ignore existing ones we don't set.
 	update.Labels(&vtk.Labels, newKeyspace.Labels)
 
+	// Update disk size.
+	update.KeyspaceDiskSize(&vtk.Spec.VitessKeyspaceTemplate, keyspace)
```

Comment:
You had the source right to begin with:

```suggestion
	update.KeyspaceDiskSize(&vtk.Spec.VitessKeyspaceTemplate, &newKeyspace.Spec.VitessKeyspaceTemplate)
```

Reply:
Done. Also I'm not sure why this confused me so much.